interface/github.py

```diff
@@ -105,6 +105,51 @@ def org_get_teams(self):
         except GithubException as e:
             raise GithubAPIException(e.data)
 
+# ---------------------------------------------------------------
+# --------------- methods related to team members ---------------
+# ---------------------------------------------------------------
+
+    def list_team_members(self, team_id):
+        """Return a list of users in the team of id team_id."""
+        try:
+            team = self.github.get_team(team_id)
+            return map(lambda x: x, team.get_members())
```

Comment:
This does not return a list. Instead, this returns a Map iterator (or a map object, whatever you like to call it). The difference between the two is that you cannot index a Map object. This is a Python 3 thing. Please convert it to a list:

```python
list(map(lambda x: x, team.get_members))
```

Reply:
didn't know that. Good call